src/status_im2/common/bottom_sheet/style.cljs

```diff
@@ -47,8 +47,9 @@
 (defn selected-item
   [theme window-height sheet-height {:keys [top]}]
   {:position          :absolute
-   :bottom            10
-   :max-height        (- window-height sheet-height top)
+   :bottom            8
+   ;; we minus 8 on the max-height since the bottom has a margin of 8
```

Comment:
How about using a variable for `bottom` instead of hardcoding `8` in both spots? This way, we won't need this comment, and it'll keep our UI safe in case we tweak `bottom` down the road.